.github/workflows/ci.yml

```diff
@@ -206,6 +206,11 @@ jobs:
       - name: Install additional requirements 
         env:
           GITHUB_REPOSITORY: ${{ github.repository }}
+          MATRIX_PHPUNIT: ${{ matrix.phpunit }}
```

Comment:
We either don't need this as an environment variable, or we should also be setting other matrix things to environment variables. We use `matrix.phpunit_suite` directly below, for example.

If there's some reason we need to treat this differently than `matrix.phpunit_suite`, please add a code comment here so this isn't confusing next time we look at it.

Reply:
Changed to just using `${{ matrix.phpunit }}` directly